src/js/abstract/hocs/with-update.js

```diff
@@ -43,40 +38,7 @@ const withUpdate = Base =>
             lifecycleLogger(this.logLifecycle)(`\n<-> apply getter/setter for ${key} by _${attr}`);
           }
 
-          if (PROPERTY_WHITELIST.indexOf(key) === -1 && hasKey) {
-            throw new PropertyExistsException(key, this);
-          }
-
           this._hasKeys[key] = hasKey;
-
-          // @todo: may we should allow deletion by setting configurable: true
-          Object.defineProperty(this, key, {
```

Comment:
That introduced more problems that it solved

Reply:
Though we could still log a warning, saying hay look, this is a standard HTML attribute/DOM property already in the interface...

Reply:
Anyway by namespacing it we shouldn't any problem with it...